PasswordChecker/PasswordChecker.py

```diff
@@ -0,0 +1,17 @@
+import requests
+import hashlib
+
+string = input('Enter the password')
```

Comment:
This line will hang our continuous integration server Travis CI + pytest.  Please read CONTRIBUTING.md.

Creating a new directory is not allowed for a single algorithm -- this does not fit with the style for the rest of the repo.

directory names and file names should be in _snake_case as discussed in CONTRIBUTING.md.

Algorithms in this repo need to be functions or classes with type hints and doctests as discussed in CONTRIBUTING.md.